### ISSUE TITLE
Figure problem

### ISSUE THREAD

**Issue by leesocool** [JJisbug/RDP-GAN#3]
Hi, I was reading this article recently, and I think there is an error in Figure 4 of the article, and I understand from Figure 6 that the classification accuracy when the total privacy budget is 5 is significantly higher than it is 0.5. But in Figure 4, the picture quality when the total privacy budget is 5 is not as good as when it is 0.5.

**Comment by JJisbug** [comment]
Yes, the current _arxiv_ version is a very old one, and we have corrected this error and added more details. As the current version is under review, we will upload a well edited version when the manuscript is accepted. Thanks

**Comment by leesocool** [comment]
Thank you for your reply, hope the manuscript will be accepted successfully.

**Comment by JJisbug** [comment]
> 

Thanks, and please find the newest version of the manuscript in the attachment.
[bare_jrnl_compsoc.pdf](https://github.com/JJisbug/RDP-GAN/files/9751285/bare_jrnl_compsoc.pdf)
